src/components/plots/errorBarPlot.ts

```diff
@@ -0,0 +1,47 @@
+module Plottable {
+export module Plot {
+  export class ErrorBar<X> extends AbstractXYPlot<X,number> {
+
+    protected _isVertical: boolean;
+    private _defaultStrokeColor: string;
+
+    /**
+     * Constructs an ErrorBarPlot.
+     *
+     * Error bar plots are used to show variability in a reported measurement on a plot.
+     * They are intended to be merged on top of other sorts of plots.
+     *
+     * @constructor
+     * @param {QuantitativeScale} xScale The x scale to use.
+     * @param {QuantitativeScale} yScale The y scale to use.
+     */
+    constructor(xScale: Scale.AbstractQuantitative<X>, yScale: Scale.AbstractQuantitative<number>, isVertical = true) {
+      super(xScale, yScale);
+      this.classed("error-plot", true);
+      this._defaultStrokeColor = new Scale.Color().range()[1];
```

Comment:
Is there a reason you wanted to use the 2nd color in the range rather than a specified color?